chalicelib/checks/wfr_checks.py

```diff
@@ -1514,3 +1513,123 @@ def margi_start(connection, **kwargs):
 
     action = wfr_utils.start_tasks(missing_runs, patch_meta, action, my_auth, my_env, start, move_to_pc=True, runtype='margi')
     return action
+
+@check_function(lab_title=None, start_date=None)
+def bed2multivec_status(connection, **kwargs):
+    """Searches for bed files states types that don't have bed2multivec
+    Keyword arguments:
+    lab_title -- limit search with a lab i.e. Bing+Ren, UCSD
+    start_date -- limit search to files generated since a date formatted YYYY-MM-DD
+    run_time -- assume runs beyond run_time are dead (default=24 hours)
+    """
+    start = datetime.utcnow()
+    check = CheckResult(connection, 'bed2multivec_status')
+    my_auth = connection.ff_keys
+    check.action = "bed2multivec_start"
+    check.brief_output = []
+    check.full_output = {}
+    check.status = 'PASS'
+
+    # check indexing queue
+    env = connection.ff_env
+    indexing_queue = ff_utils.stuff_in_queues(env, check_secondary=True)
+    if indexing_queue:
+        check.status = 'PASS'  # maybe use warn?
```

Comment:
I'd ask Burak maybe on what he would prefer - in my mind `WARN` makes more sense as `PASS` implies the check ran successfully when in fact it really did not.